src/components/EpisodesNav/renderUtils.jsx

```diff
@@ -37,10 +37,28 @@ function renderPaginationLink(data, index){
 
 	const episodeId = UTILS.convertToSafeString(title, "-");
 
-	const isActive = index === activeIndex;
+	const isActive 	= index === activeIndex;
+
+	let opacity = 1;
+	if(!isActive){
```

Comment:
For the purposes of A/B-ing it, I think it works better to be like

```
let opacity;
if(isActive) opacity = 1;
else {
    ...
    opacity = 1 - (fadeOffset  * decayRate);
}
```

rather than have a default value and the secretly overriding it in an if statement. by putting a `let` above an `if/else` it makes it super-clear what the role of that `if/else` is, and also that the `opacity` is about to be definied right here and now.